src/bin/pgcopydb/schema.h

```diff
@@ -354,10 +354,11 @@ typedef enum
 	DATA_SECTION_FILTERS,
 	DATA_SECTION_BLOBS,
 	DATA_SECTION_VACUUM,
-	DATA_SECTION_ALL
+	DATA_SECTION_ALL,
+	DATA_SECTION_NAMESPACES
 } CopyDataSection;
 
-#define DATA_SECTION_COUNT (DATA_SECTION_ALL + 1)
+#define DATA_SECTION_COUNT (DATA_SECTION_NAMESPACES + 1)
```

Comment:
Please keep DATA_SECTION_ALL as the last entry in the list, and then DATA_SECTION_COUNT definition does not need to change either. I think I would insert DATA_SECTION_NAMESPACES just before DATA_SECTION_TABLE_DATA in the enum.